src/main/java/org/jenkinsci/plugins/gwt/JobFinder.java

```diff
@@ -12,11 +18,43 @@
 public final class JobFinder {
   private JobFinder() {}
 
-  public static List<GenericTrigger> findAllJobsWithTrigger() {
+  public static List<GenericTrigger> findAllJobsWithTrigger(String queryStringToken) {
+
     List<GenericTrigger> found = new ArrayList<>();
+    // Impersinate to get all jobs even without read grants
+    SecurityContext orig = ACL.impersonate(ACL.SYSTEM);
```

Comment:
Should be changed to `ACL.as` to avoid using deprecated API. http://javadoc.jenkins-ci.org/hudson/security/ACL.html#impersonate-org.acegisecurity.Authentication-